modules/repofiles/action.go

```diff
@@ -19,6 +22,16 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
+const (
+	secondsByMinute = float64(time.Minute / time.Second) // seconds in a minute
```

Comment:
isnt souch const's in *time* package?

Reply:
-> just asking didn't had a look jet ...

Reply:
Not one I'm aware of 😉